### ISSUE TITLE
Ask for confirmation when moving a post to the trash

### ISSUE THREAD

**Issue by mrfoxtalbot** [WordPress/gutenberg#19125]
Gutenberg will not ask for confirmation when clicking on the "Move to Trash Button", making it easy for a user to delete a post accidentally.

![](https://d.pr/i/VZWHcW+)

The "1 post sent to the Trash." message that appears after this is not too visible either.

![](https://d.pr/i/P3GaCF+)

**Describe the solution you'd like**
I think we need an intermediate step to make sure users did not click on the "Move to Trash" button by accident. 

**Describe alternatives you've considered**
I would like to see a confirmation message, asking the user to confirm that the indeed wat to delete that post. Something like this.

![](https://d.pr/i/Sx0Mwu+)

**Comment by MichaelArestad** [comment]
Hey! Thanks for reporting this. This was discussed today [in Slack](https://wordpress.slack.com/archives/C02S78ZAL/p1576603443057200). There's definitely room for improvement here.

The likely solution is similar to what you proposed with a secondary confirmation step of some sort.

That said, I believe this is a lower priority right now for a couple reasons. It's an action that can be undone. It's fairly obvious when it happens as the user is directed away from the editor to Posts or Pages.

Again, this is definitely a flow with room for improvement, but we have limited design resources to tackle this at the moment so we need to prioritize our efforts.

**Comment by mrfoxtalbot** [comment]
I love that idea @richtabor!

**Comment by jodamo5** [comment]
I'd definitely like this confirmation button to be added.  On my Wordpress installation, for some reason the "Move to Trash" button appears about 1 sec after the rest of the page has loaded.  I tried to click on "Revisions" but just as I did the "Move to Trash" button appeared, pushing the Revisions button down inadvertently causing the post to be moved to trash. I've also had a client who accidentally moved a post to trash without realising. A confirmation step would be very useful.
